src/aws_encryption_sdk/internal/formatting/encryption_context.py

```diff
@@ -64,11 +70,14 @@ def _serialize_encryption_context_list(encryption_context_list, serialized_conte
         # we check to make sure that we return the right type of error message for an overly long key or value
         except struct.error as struct_error:
             message = str(struct_error)
+            # a key or value were too long and struct gave us the expected error
```

Comment:
I don't think this comment clarifies anything -- what do you think?

Reply:
deleted and other unhelpful commenting